backend/src/main/java/com/cruru/process/service/ProcessService.java

```diff
@@ -101,6 +103,17 @@ private void validateProcessCount(List<Process> processes) {
         }
     }
 
+    @Transactional
+    public ProcessResponse update(long processId, @Valid ProcessUpdateRequest request) {
```

Comment:
```suggestion
    public ProcessResponse update(@Valid ProcessUpdateRequest request, long processId) {
```
추후 수정할 때 용이하도록 파라미터 순서를 이렇게 변경해 보는 건 어떠신가요?